packages/phone-number-privacy/src/match-making/get-contact-matches.ts

```diff
@@ -63,3 +67,7 @@ function hasValidUserPhoneNumberParam(requestBody: any): boolean {
 function hasValidContractPhoneNumbersParam(requestBody: any): boolean {
   return requestBody.contactPhoneNumbers && Array.isArray(requestBody.contactPhoneNumbers)
 }
+
+function isBodyReasonablySized(requestBody: any): boolean {
+  return JSON.stringify(requestBody).length <= REASONABLE_BODY_CHAR_LIMIT
```

Comment:
Not a big deal but I wonder if there's a way to check this without re-stringifying the request body, since it gets parsed automatically by express.

Reply:
Good question, I wasn't able to find a way to do this using the request object from Express 🤔